RubeesEat.IntegrationTests/SeleniumTests/RubeesEatSeleniumTests.cs

```diff
@@ -27,38 +27,76 @@ public void CorrectInput()
         
         page.SetNewDesciptionText("Test");
         page.SetNewTotalPriceText("5");
-        page.ClickAddPerson();
-        var persons = page.GetPersonAmounts();
-        persons[0].SetAmountForPerson("5");
+        var person = page.ClickAddPerson();
+        person.SetAmountForPerson("5");
         page.ClickSplitBill();
         
         Assert.That(page.GetPersonAmountAndNames(), Is.Empty);
     }
     
+    [Test]
+    public void InvalidTotalAmount()
+    {
+        var page = Start<SplitBillPageObject>();
+        
+        page.SetNewDesciptionText("Test");
+        page.SetNewTotalPriceText("e");
+        var person = page.ClickAddPerson();
+        person.SetAmountForPerson("5.5");
+        page.ClickSplitBill();
+        
+        Assert.That(page.GetErrorMessage(), Is.EqualTo("Please enter a valid number for total amount."));
+    }
+    
+    [Test]
+    public void InvalidAmountPerPerson()
+    {
+        var page = Start<SplitBillPageObject>();
+        
+        page.SetNewDesciptionText("Test");
+        page.SetNewTotalPriceText("5.5");
+        var person = page.ClickAddPerson();
+        person.SetAmountForPerson("e");
+        page.ClickSplitBill();
+        
+        Assert.That(page.GetErrorMessage(), Is.EqualTo("Amounts are not valid."));
+    }
+    
     [Test]
     public void WrongAmount()
     {
         var page = Start<SplitBillPageObject>();
         
         page.SetNewDesciptionText("Test");
         page.SetNewTotalPriceText("10");
-        page.ClickAddPerson();
-        var persons = page.GetPersonAmounts();
-        persons[0].SetAmountForPerson("5");
+        var person = page.ClickAddPerson();
+        person.SetAmountForPerson("5");
         page.ClickSplitBill();
         
         Assert.That(page.GetErrorMessage(), Is.EqualTo("The sum of individual amounts doesn't match the total bill amount."));
     }
+    
+    [Test]
+    public void NegativeAmount()
+    {
+        var page = Start<SplitBillPageObject>();
+        
+        page.SetNewDesciptionText("Test");
+        page.SetNewTotalPriceText("-10");
+        var person = page.ClickAddPerson();
+        person.SetAmountForPerson("-10");
```

Comment:
amounts should add up to 0

Reply:
-10 is the user input. The second input will be changed to 10 in EntryLine. So it already adds up to 0